### ISSUE TITLE
Show route hints on the website?

### ISSUE THREAD

**Issue by osmuser63783** [openstreetmap/openstreetmap-website#5275]
### Problem

GraphHopper doesn't just output directions, it also outputs details (referred to as "route hints" or "path details" in the documentation) about things like dangerous sections.

What are your thoughts on showing these on the osm.org website, in addition to the directions?

### Description

Examples of "route hints" include
- Route includes potentially dangerous trunk roads or worse
- Get off the bike and push
- Route includes steep sections
- Route includes steps
- Route includes fords
- Route crosses a country border
- Route includes ferries
- Route has tolls
- Route with potential access restrictions

These could be shown in an info box above the directions.

Why? For mappers, it's useful feedback that their tags are being looked at by data consumers. For non-mappers, I think strategically it's important for OSM that osm.org shows off the cool things that can be done with OSM data, to attract them to contribute to OSM. When people who aren't really familiar with OSM are shown nonsensical or dangerous routes, I am worried it will turn them off OSM because they don't understand that "under the hood" there's a lot more going on.

### Screenshots

<img width="1199" alt="Screenshot 2024-10-20 at 18 42 29" src="https://github.com/user-attachments/assets/f24d0ee2-d3ad-460d-8101-764b706ef85e">

<img width="1199" alt="Screenshot 2024-10-20 at 18 44 13" src="https://github.com/user-attachments/assets/3279c5ab-3273-4553-a617-94545ca38931">

**Comment by mmd-osm** [comment]
FYI: Since it doesn't really make sense to replicate each and very routing UI feature on the osm.org repo, we're looking to add a direct link to various routing engines in #5224

**Comment by tomhughes** [comment]
Non-mappers are not our target audience so only benefits to mappers matter here.
